project/plugins.sbt

```diff
@@ -8,3 +8,4 @@ addSbtPlugin("de.heikoseeberger" % "sbt-header" % "3.0.2")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
 addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.14")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.9")
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.6.1")
```

Comment:
Looks like this mima is reporting a bunch of signature problem.
I'm guessing most are false positives or internal changes that we can ignore.

Reply:
Hi @eed3si9n sure!

Reply:
Hi @eed3si9n let me know wdyt!